hphp/hack/test/typecheck/xhp_modifier/xhp_modifier_no_colon.php

```diff
@@ -0,0 +1,6 @@
+<?hh // strict
+
+// The colon in the XHP class name is not allowed.
+xhp class :foo {}
+
+xhp class foo:bar {}
```

Comment:
This is slightly misleading: we're allowing colons in XHP class names, but not leading colons. How about something like this?

```
// XHP classes do no allow leading colons in class names.
xhp class :foo {}

// Colons elsewhere are fine.
xhp class foo:bar {}
```